serializations.md

```diff
@@ -761,6 +762,14 @@ The channel off-chain transaction is not included directly in the transaction tr
 ]
 ```
 
+version 2, from Fortuna release
```

Comment:
I see:
* [Definition in code](https://github.com/aeternity/aeternity/blob/3cad5f6848198c7931cbec214c6066d8b1117868/apps/aechannel/src/aesc_offchain_tx.erl#L198L201).
* [Relation to protocol version](https://github.com/aeternity/aeternity/blob/3cad5f6848198c7931cbec214c6066d8b1117868/apps/aechannel/src/aesc_offchain_tx.erl#L246).

Ok.